Library/Formula/unifdef.rb

```diff
@@ -13,6 +13,6 @@ def install
   end
 
   test do
-    system "echo '' | #{bin}/unifdef"
+    system "echo", "''", "| #{bin}/unifdef"
```

Comment:
I think this change breaks the test. Putting the pipe inside a quote with other characters is going to make that a single token. And the multi-argument form of `system()` bypasses the shell, so `|` won't be treated as special regardless, since that's a shell feature. Now instead of running `unifdef`, it's literally just echoing ` | /usr/local/Cellar/unifdef/2.11/bin/unifdef`. You can see this with the `-v` option; `brew test -v unifdef`.

If audit is giving you a hard time about the one-string `system`, you could switch to using `shell_output` and testing the output.